### PR TITLE
Style/#229 landing page upper white section css

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
upper white section, upper grey section and ~~part of~~ lower white section. 

~~In the lower white section, the title, logos and carousel are completed. I commented out the three yellow icon portion because this is the portion that is causing our page to act funny around 1041px. Need to figure out why.~~

The logos section now scroll horizontally if some of the logos are not accessible. 

ticket 229 reached the max number of PR it could attach.

### CODE REVIEW

File: src/pages/home/styles.scss [hunk 1]
```diff
@@ -104,32 +156,78 @@
...
 
-  .grey-section-text {
-    font-family: $font-sans-serif;
-    font-weight: 900;
-    font-size: 40px;
-    color: $navy-black;
-    letter-spacing: 0;
-    line-height: 70px;
-    text-align: center;
+      @include breakpoint(tablet-up) {
```
Comment (samrichca) [inline src/pages/home/styles.scss:205]: leftover code?

File: src/pages/home/styles.scss [hunk 2]
```diff
@@ -104,32 +156,78 @@
...
-    position: relative;
-    left: 140px;
-    bottom: 140px;
-    margin-bottom: -200px;
+    display: none;
+
+    @include breakpoint(tablet-up) {
+      display: block;
+      position: relative;
+      // left: 140px;
```
Comment (samrichca) [inline src/pages/home/styles.scss:223]: please remove comment

File: src/pages/home/styles.scss (hunk 1 above)
Comment (daniel-cy-lu) [reply src/pages/home/styles.scss:205]: line 205 removed

File: src/pages/home/styles.scss (hunk 2 above)
Comment (daniel-cy-lu) [reply src/pages/home/styles.scss:223]: comment removed

File: src/pages/index.js [hunk 3]
```diff
@@ -115,35 +117,53 @@ export default function HomePage() {
...
+            specialists developing big-data tools for those advancing the knowledge and treatment of
+            cancer.
           </h2>
         </div>
 
-        <div className="upper-grey__line">
-          <h2 className="grey-section-text">treatment of cancer.</h2>
+        <div className="upper-grey__not-mobile">
+          <div className="upper-grey__line ">
+            <h2 className="upper-grey__text">We</h2>
```
[the remark this answers is not in the document]
Comment (daniel-cy-lu) [reply src/pages/index.js:132]: oh right <span>. I forgot about it. Let me do that.

[the remark this answers is not in the document]
Comment (daniel-cy-lu) [reply src/pages/index.js]: Updated h2 to span element.

**Review by samrichca — APPROVED** [review]
looks great! LGTM!

carousel isn't accessible at the moment, looking into it.

**Pull request merged**